client/components/editor.js

```diff
@@ -0,0 +1,42 @@
+Template.editor.helpers({
+  docid: function(){
+    var doc = Documents.findOne();
+    if (doc){
+      return doc._id
+    } else {
+      return undefined
+    }
+  },
+  config: function(){
+    return function(editor){
+      // console.log(editor)
+      editor.on("change", function(cm_editor, info){
+        // console.log(cm_editor.getValue())
+        $("#viewer_iframe").contents().find("html").html(cm_editor.getValue())
+        Meteor.call("addEditingUser")
```

Comment:
:+1: will get to linting, test and CI on meteor. Still familiarising myself with the environment